app/views/katello/api/v2/subscriptions/show.json.rabl

```diff
@@ -59,10 +59,3 @@ node :activation_keys do |subscription|
     }
   end
 end
-
-node :host, :if => lambda { |sub| sub && sub.host } do |subscription|
```

Comment:
Actually thats the wrong issue.  There were two user stories here: http://projects.theforeman.org/projects/katello/wiki/HostUnification  They were P3, so weren't entered.

```

    As a developer, I want to pull in subscriptions and related hosts into the database (7)
    As a user, I expect to search for hosts by their subscriptions (.5)
```

  I'll enter those to cover this.